planet/cli/orders.py

```diff
@@ -221,6 +221,35 @@ async def create(ctx, request: str, pretty):
     echo_json(order, pretty)
 
 
+def stash_item_type(ctx, param, value):
+    ctx.obj['item_type'] = value
+    return value
+
+
+def bundle_cb(ctx, param, value):
+    item_type = ctx.obj['item_type']
+    choices = planet.specs.get_product_bundles(item_type)
+    if value is None:
+        msg = ("Choose from:\n\t{choices}").format(
+            choices=",\n\t".join(choices))
+        raise click.ClickException(f"Missing option '--{param.name}'. {msg}")
```

Comment:
@kevinlacaille will you try moving this branch into Bundle.convert? Maybe we could do away with the callback and simplify this.